src/lib/card-types.ts

```diff
@@ -95,7 +95,7 @@ const cardTypes: CardCollection = {
       [8110, 8131],
       [8132, 8151],
       [8152, 8163],
-      [8164, 8171],
+      [8164, 8171, 8173],
```

Comment:
```suggestion
      [8164, 8171],
      8173,
```

Reply:
Thanks for the suggestion, It was really helpful appreciated